src/web/browser/App.tsx

```diff
@@ -124,13 +124,15 @@ const App = ({ CAPI, NAV }: Props) => {
             <Hydrate root="nav-root">
                 <Nav pillar={CAPI.pillar} nav={NAV} />
             </Hydrate>
-            <Hydrate root="sub-nav-root">
-                <SubNav
-                    subnav={NAV.subNavSections}
-                    currentNavLink={NAV.currentNavLink}
-                    pillar={CAPI.pillar}
-                />
-            </Hydrate>
+            {NAV.subNavSections && (
+                <Hydrate root="sub-nav-root">
+                    <SubNav
+                        subNavSections={NAV.subNavSections}
+                        currentNavLink={NAV.currentNavLink}
+                        pillar={CAPI.pillar}
+                    />
+                </Hydrate>
+            )}
```

Comment:
💯 This component would never have rendered on the server if it has no subNavSections so trying to hydrate it here was a waste, thanks for fixing!